code/game/objects/structures/barricade/barricade.dm

```diff
@@ -459,30 +460,38 @@
 			repair_value = repair_materials[validSheetType]
```

Comment:
This throws runtimes if you use it next to a broken cade (plasteel in this case) with no sheet in in your off hand:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/3af5fb68-8d99-4b3f-a31e-065b71d0783b)